### ISSUE TITLE
Semi-Global - The state of the current tab is not announced

### ISSUE THREAD

**Issue by katknow** [omeka/Omeka#1142]
"Description: On the appearance page, there are three tabs that the user can navigate to. The page currently in focus is not announced to the user, but is shown visually. Currently, when the user navigates to one of the three tabs on the page, the state (e.g., ""current"", ""selected"") of the tab is not announced. All interactable elements have a proper name, role and as-needed value. This could be resolved by having the words 'current tab' announced before the name of the tab that currently has focus.

Note: This issue also occurs on other pages with tabs, such as the Simple Pages page and Add Items page.

Expected Results: When there are multiple tabs on a page, the current status of where focus is placed is announced to the user as well as shown visually. Adding aria-current=""page"" to the <a> of the current page should have screen readers announce ""current page"" with the link. 

Actual Results: The tab on which the user is currently on is not announced."

4.1.2 Name, Role, Value; Level A; WCAG 2.0

**Comment by kimisgold** [comment]
Updated the nav pattern to includes `aria-current` in [`a11y/fix-tabs`](https://github.com/omeka/Omeka/tree/a11y/fix-tabs).
